src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return sum - 20;
+  } else if (days === 7) {
+    return sum - 50;
```

Comment:
The value '7' and '50' are literals and should be replaced with constants that describe what these values represent, such as 'LONG_TERM' and 'LONG_TERM_DISCOUNT'. Also, the condition should be 'days >= 7' to cover all cases of 7 or more days.